src/main/java/seedu/expense/model/expense/Amount.java

```diff
@@ -3,30 +3,38 @@
 import static java.util.Objects.requireNonNull;
 import static seedu.expense.commons.util.AppUtil.checkArgument;
 
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
 /**
  * Represents an Expense's amount in the expense book.
  * Guarantees: immutable; is valid as declared in {@link #isValidAmount(String)}
  */
-public class Amount {
-
+public class Amount implements Comparable<Amount> {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Expense amount should only contain numbers and 1 '.', and should be in the {dollars}.{cents} format."
-                    + "{.cents} is optional but {dollars} should contain at least 1 digit.";
+            "Expense Amount restrictions:\n"
+            + "Amount Format: should only contain numbers and 1 '.', and should be in the <dollars>.<cents> format.\n"
+            + "    \".<cents>\" input is optional but <dollars> should contain at least 1 digit.\n"
+            + "Value Restrictions: -10e9 to 10e9 inclusive";
```

Comment:
Value restriction should be 0 to 10e9 since amount cannot be negative

Reply:
yup done!